apps/site/src/components/pages/wordpress/early-access-cta.tsx

```diff
@@ -0,0 +1,161 @@
+import { faCheck } from "@fortawesome/free-solid-svg-icons";
+import { inputBaseClasses, Theme, useMediaQuery } from "@mui/material";
+// Our custom TextField hides the 'endAdornment' when the 'error' prop is true
+// eslint-disable-next-line no-restricted-imports
+import TextField from "@mui/material/TextField";
+import { MouseEvent, TouchEvent, useMemo, useRef, useState } from "react";
+
+import { apiClient } from "../../../lib/api-client";
+import { Button } from "../../button";
+import { useEmailTextField } from "../../hooks/use-email-text-field";
+import { ArrowRightIcon, FontAwesomeIcon } from "../../icons";
+import { useEmailSubmitted } from "./email-submitted-context";
+
+const submittedEmailText = "You’re already on the waitlist as";
```

Comment:
I think this might be misread as meaning that you signed up _before_ since it appears immediately after signing up. 
```suggestion
const submittedEmailText = "You’re on the waitlist as";
```

Reply:
Do you agree on changing this @nonparibus ?